src/main/java/de/numcodex/sq2cql/model/Mapping.java

```diff
@@ -16,13 +16,13 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public final class Mapping {
 
-    private final TermCode concept;
+    private final TermCode key;
     private final String resourceType;
     private final String valueFhirPath;
     private final List<Modifier> fixedCriteria;
 
-    private Mapping(TermCode concept, String resourceType, String valueFhirPath, List<Modifier> fixedCriteria) {
-        this.concept = Objects.requireNonNull(concept);
+    private Mapping(TermCode key, String resourceType, String valueFhirPath, List<Modifier> fixedCriteria) {
```

Comment:
@alexanderkiel did we not say we would write code order termCode

Reply:
In JSON it is called key. That's why I used also key here.